src/Components/Subscriptions/index.jsx

```diff
@@ -96,20 +121,44 @@ const Subscriptions = () => {
         classes={classes}
         onClose={() => setAddForm(false)}
       />
+      <div className={styles.inputSearch}>
+        <Input
+          name="Search Subscription"
+          labelText="Search"
+          placeholder="Search Subscription"
+          value={searchTerm}
+          change={(event) => setSearchTerm(event.target.value)}
+        />
+      </div>
       <div className={styles.containerContent}>
-        <Table className={subscriptions.table} data={subscriptions} />
+        <Table className={subscriptions.table} data={filteredSubscriptions} />
+      </div>
+      <div className={styles.buttonContainer}>
+        <Button
+          classNameButton="submitButton"
+          text="+ Add New"
+          action={() => {
```

Comment:
We don't need this action any more, because we're using another route to renderize the form, you can just take it out, and use the  `<Link>`